spec/fixtures/Puppetfile_test

```diff
@@ -0,0 +1,12 @@
+mod 'puppet-hiera',
+  git: 'https://github.com/voxpupuli/puppet-hiera.git',
+  tag: '6.0.0'
+
+mod 'puppet-hiera',
+  git: 'https://github.com/voxpupuli/puppet-hiera.git',
+  tag: 'v6.0.0'
+
+mod 'puppet-hiera',
+  git: 'https://github.com/voxpupuli/puppet-hiera.git',
+  tag: 'abc'  
+  
```

Comment:
this is an unneeded newline with whitespace followed by a line without a `\n`? Interesting combination :D Clean you fix that as well?